src/Asana/Dispatcher/OAuthDispatcher.php

```diff
@@ -39,10 +40,23 @@ public function fetchToken($code)
         $params = array('code' => $code, 'redirect_uri' => $this->redirectUri);
         $result = $this->oauthClient->getAccessToken(OAuthDispatcher::$TOKEN_ENDPOINT, 'authorization_code', $params);
         $this->accessToken = $result['result']['access_token'];
+        $this->refreshToken = $result['result']['refresh_token'];
         $this->authorized = !!$this->accessToken;
         return $this->accessToken;
     }
 
+    public function refreshToken()
```

Comment:
maybe we should call this `refreshAccessToken()` since we're using "refresh" as a verb? Based on its name it sounds like it might be "getting the refresh token".

Also, should we add comments to this public method?